src/nidcpower/metadata/functions_addon.py

```diff
@@ -2,6 +2,17 @@
 # Changes to the metadata should be made here, because functions.py is generated thus any changes get overwritten.
 
 functions_override_metadata = {
+    "ConfigureLCRCustomCableCompensation": {"codegen_method": "no"},
```

Comment:
please add TODO pointing to Issue for adding these

Reply:
I've filed an issue for this and added a TODO comment referencing that issue in the latest commit.